apps/sms/views/conversation/js/conversation.js

```diff
@@ -265,6 +266,8 @@ var ConversationView = {
 
     // Bound methods to be detachables
     this.onMessageTypeChange = this.onMessageTypeChange.bind(this);
+
+    MessagingClient.init();
```

Comment:
Yeah you are right, MessagingClient/service are not necessary for startup path (and even not necessary for conversation view startup)